src/logic/base.py

```diff
@@ -0,0 +1,42 @@
+from textwrap import indent
```

Comment:
surely this should all go in `__init__.py`?

Reply:
If you mean everything in the module I just like to keep `__init__` clear from the actual code and just build package's api there. It's the same as with `__init__` and `main` in `pydantic`. I can call this module `common` or `main` if you'd prefer that.

Reply:
but this package is much smaller than pydantic, and it has a single entry point, not many. Hence I think all this logic should just go into `__init__.py`.

Reply:
ok

Reply:
Updated

Reply:
Updated

Reply:
Actually, I would argue to introduce `base` module back. As now `__init__` imports `prs` and will import `issues` and those are importing `models` and this introduces a circular import. Basically, I think this refactoring brings `logic` into the league of packages and we need to follow the rules 🤷🏻